src/lamp_py/ingestion_tm/jobs/parition_table.py

```diff
@@ -149,26 +123,30 @@ def dates_to_export(self, tm_db: MSSQLManager) -> List[int]:
     def run_export(self, tm_db: MSSQLManager) -> None:
         table_columns = ",".join([col.name for col in self.export_schema])
 
-        logger = ProcessLogger("tm_stop_crossing_export")
```

Comment:
if `tm_db.write_to_parquet`fails, this process was still running as if nothing was wrong, making debugging difficult because we can't see what subclass has the problem. now it'll be caught here with more debugging information.